sdk/ai/ai-inference-rest/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.2 (Unreleased)
```

Comment:
we have a bot that increments package version after a package is released. Why is it added now?

Reply:
Will remove, got an obscure changelog verification fail i'm trying to fix

Reply:
Looks that we pack this file too, so it's better to remove it now.